plugins/PrivacyManager/templates/askingForConsent.twig

```diff
@@ -26,7 +26,7 @@
 
    {% if consentManagerName %}
     <div piwik-content-block content-title="{{ 'PrivacyManager_ConsentManager'|translate|e('html_attr') }}" class="privacyAskingForConsent">
-        <p> {{ 'PrivacyManager_ConsentManagerDetected'|translate(consentManagerName, '<a href="', consentManagerUrl, '" target="_blank" rel="noreferrer noopener">', '</a')|raw }}
+        <p> {{ 'PrivacyManager_ConsentManagerDetected'|translate(consentManagerName, '<a href="' ~ consentManagerUrl ~ '" target="_blank" rel="noreferrer noopener">', '</a')|raw }}
```

Comment:
@bx80 We should try to avoid using unneeded placeholders where possible. Especially breaking up html tags into multiple placeholders is not a good idea, as it might happen in translations that the placeholders are placed in incorrect order by accident, which would then result in broken html.